src/help/zaphelp/contents/start/concepts/authentication.html

```diff
@@ -216,6 +223,11 @@ <h2>See also</h2>
 			<td><a href="users.html">Users</a></td>
 			<td>for an overview of Users</td>
 		</tr>
+		<tr>
+			<td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
+			<td><a href="../../../start/concepts/anticsrf.html">Anti-CSRF tokens</a></td>
```

Comment:
This link is broken (it can be just `anticsrf.html`).

Reply:
oh, apologize me